net-utils/src/lib.rs

```diff
@@ -381,29 +381,57 @@ pub fn is_host_port(string: String) -> Result<(), String> {
     parse_host_port(&string).map(|_| ())
 }
 
+#[derive(Clone, Debug, Default)]
```

Comment:
1.17 toolchain's clippy is buggy and ignores `allow(clippy::derivable_impls)` here, so i went ahead and derived it since there shouldn't be much of a maintenance window on the 1.17 branch